test/tests/pbs_smoketest.py

```diff
@@ -562,12 +562,11 @@ def test_submit_job_with_script(self):
         """
         Test to submit job with job script
         """
-        a = {ATTR_rescavail + '.ncpus': '2'}
-        self.server.manager(MGR_CMD_SET, NODE, a, id=self.mom.shortname)
         j = Job(TEST_USER, attrs={ATTR_N: 'test'})
-        j.create_script('sleep 120\n', hostname=self.server.client)
+        j.create_script('sleep 5\n', hostname=self.server.client)
```

Comment:
I'd leave the job alone and then just delete it after you do what you want instead of checking for it to end.  On slow machines you might miss the job running in 5 seconds.

Reply:
What I meant by leaving the job alone is to revert it to sleep 120 seconds.  5s is still too short of a sleep.